.grunt-config/sass.js

```diff
@@ -88,6 +88,13 @@ const sass = {
 				expand: true,
 				cwd: 'assets/dev/scss/admin',
 				src: 'wc-product-editor.scss',
+				dest: 'assets/css/conditionals',
+				ext: '.css',
+			},
+			{
+				expand: true,
```

Comment:
You probably can delete this.

All files in that folder are automatically added to the css folder.

![Screenshot 2024-08-27 at 14 53 22](https://github.com/user-attachments/assets/602e1ff6-86a3-4de1-a6a0-09f42f7cb626)